### DISCUSSION TITLE
Is it possible to stop emails appearing in SENT items when using SMTP?

### DISCUSSION THREAD

**Discussion by kreativejuices** [PHPMailer/PHPMailer#2742] [General]
Hi,

I'm wondering if its possible to somehow stop emails from appearing in the sent items box on outlook when using SMTP via a website script. 

Or is this something that can simply not be configured and is totally controlled by the mail server and not phpmailer?

Thanks

**Comment by Synchro** [comment]
Indeed this is entirely down to configuration of the sending server. You may notice for example that gmail doesn't do this, which is why the gmail example code includes a snippet showing how to upload a sent message over IMAP.

**Reply by kreativejuices** [reply]
Thanks!
